StRoot/StTriggerUtilities/StDSMUtilities/DSMAlgo_BC101_2015.cc

```diff
@@ -28,10 +28,10 @@ void DSMAlgo_BC101_2015::operator()(DSM& dsm)
 
   // ACTION:
 
-  unsigned int jpx;			// East (-1 < eta < 0)
```

Comment:
This change presumes that jpx y and z can be negative.  Are these jetpatch sums?  If so, can they actually be negative? Looking at the code does not make that obvious to me.  You only increment by values which I presume are all expected to be positive.